spec/features/document_uploads_at_info_section_spec.rb

```diff
@@ -26,17 +26,17 @@
   context "when an admin uploads a document" do
```

Comment:
I think we could rename this file as `documents_uploads_at_document_page_spec.rb`, since the `Info` and `Document` now have their own dedicated pages